rustotp/src/lib.rs

```diff
@@ -1,41 +1,13 @@
 extern crate rusthotp;
+use rusthotp::{HotpOutput, hotp};
 
-use rusthotp::{ HotpOutput, hotp };
+extern crate byteorder;
+use byteorder::{BigEndian, ByteOrder};
 
```

Comment:
Just wondering what happened to the docstring here? Did you find it superfluous?